### ISSUE TITLE
Can you share the dataset link ?

### ISSUE THREAD

**Issue by irfanrah** [danielkonecny/sports-poses-recognition#1]
Hello, Thank you for your great code. However, could you send me your dataset ? and how to use python3 src/model/Encoder.py in the dataset ? 

Thank you

**Comment by danielkonecny** [comment]
Hello and thank you for your interest.

You can find example of the dataset [here](https://drive.google.com/drive/folders/11hsWDKtNw8VVH17k62QQypx4_DG_rzcU?usp=sharing) (accessible without any permission). This should serve well enough for knowing the dataset structure and providing the model with your own data. The dataset is structured into scenes, each one recorded from 3 viewpoints (if you want to provide a different number of viewpoints, you have to adjust the `DatasetHandler.py`'s code, let me know if you want further details about this).

The whole dataset can be found [here](https://drive.google.com/drive/folders/1rf7VUufuO-bRXpsxhfm7E_Cf6nO6-974?usp=sharing) (accessible only with permission). Please, request for the permission and I'll be happy to grant it, I just don't want the dataset to be accessible publicly since it contains images of myself.

You can launch the training via this command: `python3 src/model/Encoder.py "dataset/self_supervised" --fit_epochs=10 --margin=0.1 --batch_size=64 --validation_split=0.1 --encoding_dim=64 --verbose`. 
Above, I'm sharing the `self_supervised` folder, adjust the path in the command according to your needs. Other command line parameters can be found via `--help` for each of the Python scripts. Launch everything from the project root folder.

I have to mention that the dataset was unfortunately not variable enough to provide for ideal results of the self-supervised training with time-contrastive network. I further discuss the results and their consequences in my Master's Thesis [here](https://www.fit.vut.cz/study/thesis-file/24543/24543.pdf) in Section 5.1.2. However, if a more variable dataset or an easier task were presented, I expect the model to work just fine, there should not be any implementation or design flaws.

I have the dataset also labeled with 2 different sets of labels for classification purposes (`Recognizer.py`) when the self-supervised pre-training is done. I can share these with you as well, just let me know if interested. If you have any further questions or requests, I am happy to answer them.

**Comment by irfanrah** [comment]
Hi Daniel,

Thank you for your great and detail reply. I am currently request the whole dataset and hope you can accept it. No worries, I just need to run your code and won't share your dataset to others. My purpose for using this because I want to compare our self-supervised code.
